src/hazelcore/UHTyp.re

```diff
@@ -56,7 +56,7 @@ let contract = (ty: HTyp.t): t => {
   and contract_to_seq = (~parenthesize=false, ty: HTyp.t) => {
     let seq =
       switch (ty) {
-      | Hole => Seq.wrap(Hole)
+      | Unknown(_) => Seq.wrap(Hole)
```

Comment:
write out the three cases instead of wildcard here since this wouldn't always just go to Hole if we had unique IDs